eng/Versions.props

```diff
@@ -34,7 +34,7 @@
     <VisualStudioEditorPackagesVersion>16.10.44</VisualStudioEditorPackagesVersion>
     <ILAsmPackageVersion>5.0.0-alpha1.19409.1</ILAsmPackageVersion>
     <ILDAsmPackageVersion>5.0.0-preview.1.20112.8</ILDAsmPackageVersion>
-    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.10.1202</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
+    <MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>16.11.2</MicrosoftVisualStudioLanguageServerProtocolPackagesVersion>
```

Comment:
can this go in main?

Reply:
Yep, we ship a copy of this dll, so we can change it to whatever version we need (as long as it doesn't bring in other dependency updates)